COVIDWatch iOS/Utility/UIWindow+ViewControllerUtility.swift

```diff
@@ -16,15 +16,15 @@ extension UIApplication {
         }
         return nil
     }
-    
+
     var topViewController: UIViewController? {
         #if AppExtension
             return nil
         #else
-            return topViewController(keyWindow)
+            return topViewController(windows.first)
```

Comment:
Updated to silence a warning about using keyWindow (there can be multiple windows in iOS 13+).